__tests__/app.test.js

```diff
@@ -107,3 +107,25 @@ describe("GET /api/reviews/:review_id", () => {
       });
   });
 });
+
+
+describe("GET /api/users", () => {
+    test("Get request to /api/users responds with array of objects with designated ", () => {
```

Comment:
I would reword this as it not clear what it responds with, what is an array of objects with designated?